defaults/main.yml

```diff
@@ -17,6 +17,10 @@ node_exporter_binary_arch: "amd64"
 # node_exporter_install_from_binary is true.
 node_exporter_group: "node_exporter"
 
+# ProcessType to use in the Mac launchd plist file. Please refer to `man 5 launchd.plist`
+# for an explanation of different values.
+node_exporter_process_type: "Background"
```

Comment:
Shouldn't we reflect that this variable is only for the Mac launchd plist in the variable name?

Reply:
Sure. Fixed in d45a509.